policy/joining-governance.md

```diff
@@ -0,0 +1,30 @@
+# Joining Electron Governance
+
+This documents the process for becoming a member of Electron Governance. Being a member of Electron Governance means you are actively a member of at least one Working Group.
+
+## Requirements
+
+* A new member is approved by their prospective Working Group according to their rules for membership. Approval is recorded in meeting notes.
+* A new member has a GitHub account
+* A new member is a guest on Electron's Slack
+
+## Steps
+
+1. Submit a the Workflow form for "New Governance Member" in #wg-community-safety channel.
+
+2. Update the Working Group README file to include the new member.
+
+3. An Electron GitHub owner needs to invite the new member to the Electron Org in GitHub. New member needs to accept this.
+
+4. A Google account is requested through the "Request @electronjs.org" shortcut in #access-request Slack channel.
+
+5. A Google account is provided to the new member.
```

Comment:
```suggestion
5. Approve the Google account request to the new member.
```

Reply:
This step here isn't just an approval. The google account must be made in order for the magic to happen. I'll reword this to be more clear about it.